keeper/state.go

```diff
@@ -32,6 +32,7 @@ import (
 )
 
 // GetChain is a helper function for retrieving the local Wormhole Chain ID.
+// TODO: maybe better renaming GetChainID
```

Comment:
lets either rename it or not (i'm indifferent), but either way we should remove the TODO before merging